lenses/mke2fs.aug

```diff
@@ -39,6 +39,9 @@ let empty = IniFile.empty
     for booleans, so list all the recognized values *)
 let boolean = ("y"|"yes"|"true"|"t"|"1"|"on"|"n"|"no"|"false"|"nil"|"0"|"off")
 
+(* View: fspath *)
+let fspath = /[^ \t\n"]+/
```

Comment:
I wonder if we should modify `Rx.fspath` to this. That'd be a breaking change though...

Reply:
Yeah, IMHO it is too risky to change it for every user of `Rx`.